src/api/app/views/webui/request/_request_header.html.haml

```diff
@@ -51,6 +59,11 @@
         %mark.text-dark.alert-warning.ps-2.pe-2.text-nowrap{ title: bs_request.accept_at }
           auto accepted
           = render TimeComponent.new(time: bs_request.accept_at)
+
+  - if build_status_needs_attention?(buildresults) && supported_actions.count > 1
+    %i.fas{ class: build_status_attention_icon(buildresults) }
+    Some action need attention
```

Comment:
If you accept my previous comment, this could be simplified. There is no need to pass the `buildresults` to the helpers.

Reply:
You'll need to send the model anyway, but I see your point.